### DISCUSSION TITLE
Regarding running LVS on openroad.

### DISCUSSION THREAD

**Discussion by its-rakib** [The-OpenROAD-Project/OpenROAD-flow-scripts#867] [General]
I can't find any .lylvs file for the lvs run. Can anyone help me with that ? and it would be helpful if you can provide some manual or instruction to run lvs. Thank you

**Comment by vijayank88** [comment]
As of now calibre has support for running DRC/LVS check for ASAP7. For KLayout still work in progress.
Follow-up discussion here: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/discussions/854
